src/dcp/dcp.c

```diff
@@ -150,6 +151,7 @@ int main(int argc, char** argv)
         {"daos-preserve"        , required_argument, 0, 'D'},
         {"input"                , required_argument, 0, 'i'},
         {"chunksize"            , required_argument, 0, 'k'},
+        {"copy-xattrs"          , required_argument, 0, 'e'},
```

Comment:
minor nit - both tar and rsync use "--xattrs" (also short option '-X' for rsync) for this option.  Any reason not to be consistent?

Reply:
Yes, --xattrs / -X would be better